backend/.env.example

```diff
@@ -1,16 +1,38 @@
-# Select your video service provider; the default is vonage.
-VIDEO_SERVICE_PROVIDER='vonage or opentok'
-# If you're using the Vonage Unified Platform, enter your credentials here.
-VONAGE_APP_ID=vonage-app-id
-VONAGE_PRIVATE_KEY=vonage-app-id
-# If you're using the legacy OpenTok Platform, enter your credentials here.
-OT_API_KEY=api-key
-OT_API_SECRET=api-secret
-# If you're deploying to Vonage Cloud Runtime, enter your port here; the default is 3345.
-VCR_PORT=vcr-port
-# If you're implementing the jira service for customer feedback, enter your credentials here.
-JIRA_URL=jira-url
-JIRA_API_URL=jira-api-url
-JIRA_TOKEN=jira-token
-JIRA_PROJECT_KEY=jira-project-key
-JIRA_COMPONENT_ID=jira-component-id
+# Get your Vonage Application ID and Private Key here: https://dashboard.nexmo.com/
+# If using OpenTok SDK and credentials make sure to remove this block and uncomment the block below with OT_ variables.
+
+# The default value for VIDEO_SERVICE_PROVIDER is 'opentok', make sure to change it to 'vonage' if using the Vonage environment.
+VIDEO_SERVICE_PROVIDER='vonage'
+VONAGE_APP_ID=''
+# e.g. (this is an example application Id): 
+# VONAGE_APP_ID='12d0d0-aaaa-bbb-ccc-26b780fb'
+VONAGE_PRIVATE_KEY=''
+# e.g.: (this is an example private key)
+# VONAGE_PRIVATE_KEY='----BEGIN RSA PRIVATE KEY-----
+# MIIBOAIBAAJARsF2wfXtjllRR8nnz8+CLULn0bqgZtYktJB2BdcB5bw6OYmmDVCc
+# TeTC3VXZATdSqNA6WDWCkSVinC05uYEOEwIDAQABAkArUAaYmSkAeKCO54Pl7Ert
+# 1gT+l9XU3cW+Wqhdfnsjkdjaksdjas9phpdiQXNosI60a8YyeyBUjCtQGFwbJ1Kl
+# AiEAioOWu1s5nbB6ioOXdhbW4Ovfdkjnldfjlkndfjk/njsCIQCCxRfwRVfDcC0h
+# hvuOpFzvZ870deo1/OD8j4U8jG+aCQIgXeU55qO+eODLEN6Ha+urmikc1kyQC/KP
+# aKMjV5PzfUUCIdfgdfgdERJ1K9EVwfE/5bH1E+TERb3j21UZZphjGv15AiBBs0w5
+# WRuPspPXIAHPKrjEHkUsgDZHW/V0fJWbIjJarw==6ISP6wuV0E6SRl1xKVyoYT0D
+# 8iIsCNO9GfFOx3HWN7Q1++fAAhV37
+# -----END PRIVATE KEY-----
+# '
+
+# If using OpenTok SDK (https://tokbox.com/developer/sdks/js/) remove the variables above and use these instead.
+# OT_API_KEY=api-key
```

Comment:
I would keep VIDEO_SERVICE_PROVIDER='opentok' here, especially if you plan to change the default soon, as that will break peoples app if they haven't set this to opentok